functions/go/export-terraform/terraformgenerator/templates/log-export.tf

```diff
@@ -0,0 +1,45 @@
+{{range $logsink := .LoggingLogSink}}{{ if $logsink.ShouldCreate }}
+module "logsink-{{ $logsink.GetResourceName }}" {
+  source  = "terraform-google-modules/log-export/google"
+  version = "~> 7.3.0"
+
+  destination_uri      = module.{{with or $logsink.References.BigQueryDataset $logsink.References.PubSubTopic $logsink.References.StorageBucket }}{{.GetResourceName}}{{end}}-destination.destination_uri
+  log_sink_name        = "{{ $logsink.GetDisplayName }}"
+  parent_resource_id   = {{ $logsink.GetOrganization.GetTerraformId false }}
+  parent_resource_type = "organization"
+  include_children     = true
+}
+{{end}}{{with $logsink.References.BigQueryDataset }}
+module "{{ .GetResourceName }}-destination" {
+  source  = "terraform-google-modules/log-export/google//modules/bigquery"
+  version = "~> 7.3.0"
+
+  project_id               = module.{{ .Parent.GetResourceName }}.project_id
+  dataset_name             = "{{ .GetResourceName }}"
+  log_sink_writer_identity = module.logsink-{{ $logsink.GetResourceName }}.writer_identity
+  expiration_days          = "365"
```

Comment:
Shouldn't this be pulled from the KCC object?

https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/841/files#diff-7ef292afad49686dccb3ace8549c146d96c56e144833b3773dc0ac4d7092cf70R42

(Likewise with location).

Reply:
updated to pull from KCC